grails-app/controllers/org/pih/warehouse/product/ProductController.groovy

```diff
@@ -674,6 +676,26 @@ class ProductController {
         }
     }
 
+    def showAssociatedProducts = { StockCardCommand cmd ->
```

Comment:
Why you did not use the `renderTemplate` action from `productController`?

Reply:
it seems that initially I have misunderstood the task and added similar view for **_showAssociatedProducts** in edit page as on stock card. 
Instead I only had to display all associated products on **_productSubstitutions** page and change some labels